pytest_doctestplus/plugin.py

```diff
@@ -49,13 +49,22 @@ def pytest_addoption(parser):
                      help=("Text file format for narrative documentation. "
                            "Options accepted are 'txt', 'tex', and 'rst'"))
 
+    # add ignore-glob option for old versions of pytest
+    if LooseVersion('4.3.0') > LooseVersion(pytest.__version__):
+        parser.addoption(
+            "--ignore-glob",
+            action="append",
+            metavar="path",
+            help="ignore path pattern during collection (multi-allowed).",
+        )
+
```

Comment:
I wonder if it'd be better to just not support this option on older versions of pytest since otherwise we'll get inconsistent behavior where this works for doctests but not for regular ``test_*.py`` files?